### DISCUSSION TITLE
Collapsing board categories shouldn't retain scroll position

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#50132] [Mobile]
GitHub Mobile beta

1. looking at the GitHub public roadmap in table view
2. Scroll down a section (here Q4):
    ![Screenshot_20230315-102204.png](https://user-images.githubusercontent.com/2119212/225338548-db77375e-d70d-466c-942d-e704b36c040a.png)
3. Collapse the current category
    ![Screenshot_20230315-102214.png](https://user-images.githubusercontent.com/2119212/225338557-5c232847-e4db-40d4-9551-ff50f208dca7.png)

## Expected results
Show the top of the next category (here future)

## Actual results
The scroll offset from the top of the previous category is effectively maintained within the view of the next category

Such an offset has no value, there is no correspondence between the items in the former and items in the latter category to justify jumping that amount into the next category.

**Comment by aconsuegra** [comment]
Interesting, I can't reproduce the behavior that you describe @jsoref. I have been testing in [the same project ](https://github.com/orgs/github/projects/4247/views/2) and collapsing the Q4 category scrolls me automatically to the `Future` category. We'll keep investigating.

**Reply by jsoref** [reply]
The question is does it scroll you to the *middle* of the category? That's the bug
